### PR TITLE
Add edit user view

### LINKED ISSUE #25: Add option to display details and edit a user
Add option to display user details. For now just display edit form (password, email, first name, last name, superuser status, staff status). #26 is required before this.

### PR DESCRIPTION
**Pull request by KamilSupera**
# Issue
Closes: https://github.com/team-yuml-kkks/Elimar/issues/25

# Important
Please, merge first PR #35

### CODE REVIEW

File: elimar/modules/users/views.py [hunk 1]
```diff
@@ -23,3 +23,18 @@ class UsersList(ListView):
...
+
+class UserDetailsEdit(UpdateView):
+    model = User
+    fields = [
+        "email",
+        "password",
+        "first_name",
+        "last_name",
+        "is_staff",
+        "is_active",
```
Comment (kaniak274) [inline elimar/modules/users/views.py]: Again we don't need the `is_active` please remove it :pleading_face:

File: elimar/modules/users/views.py [hunk 2]
```diff
@@ -23,3 +23,18 @@ class UsersList(ListView):
...
+        "is_superuser",
+    ]
+    template_name = "users/user_details_edit.html"
+    success_url = reverse_lazy("users-list")
```
Comment (kaniak274) [inline elimar/modules/users/views.py:40]: Shouldn't we have those in the view?
```suggestion
```

**Review by kaniak274 — CHANGES_REQUESTED** [review]

File: elimar/modules/users/views.py [hunk 3]
```diff
@@ -23,3 +23,18 @@ class UsersList(ListView):
     model = User
     paginate_by = 20
     template_name = "users/users_list.html"
+
+
+class UserDetailsEdit(UpdateView):
+    model = User
+    fields = [
```
Comment (kaniak274) [inline elimar/modules/users/views.py:30]: Shouldn't we have a form or something?

Comment (KamilSupera) [reply elimar/modules/users/views.py:30]: I guess based on model, We don't needed it https://docs.djangoproject.com/en/4.1/ref/class-based-views/generic-editing/#django.views.generic.edit.UpdateView 🤔

**Pull request merged**